lib/github-glue.ts

```diff
@@ -225,7 +225,7 @@ export class GitHubGlue {
             repo: "git",
             state: "open",
         });
-        response.data.map((pr: octokit.PullsListResponseItem) => {
+        response.data.map((pr) => {
```

Comment:
This seems to be another ESLint-related change, rather than an Octokit-related one?

Reply:
`response.data` is already correctly typed so the typing was unnecessary (and just changing 'o' to 'O' did not work 🙄).

Reply:
Yes, I understood, but I thought that this change was maybe not necessitated by switching to the new Octokit version?

Reply:
Changing `octokit` to `Octokit` still got an error so I did the most expeditious change instead of figuring out how to specify the type (it also clarifies the object is not being cast to something else).